xbmc/input/InputCodingTable.h

```diff
@@ -30,6 +30,26 @@ class IInputCodingTable
   virtual int GetType() { return TYPE_WORD_LIST; }
 
   virtual ~IInputCodingTable() {}
+  /*! \brief Called for the active keyboard layout when it's loaded, stick any initialization here
+
+      This won't be needed for most implementations so we don't set it =0 but provide a default
+      implementation.
+
+      \return true on success, false on any fatal errors
+  */
+  virtual bool Initialize() { return true; }
+
+  /*! \brief Called for the active keyboard layout when it's unloaded, stick any cleanup here
+
+      This won't be needed for most implementations so we don't set it =0 but provide a default
+      implementation.
+  */
+  virtual void UnInitialize() {}
```

Comment:
A more-correct term is "Deinitialize". "Uninitialize" is also correct as one word without camel casing.

Just some advice from a native speaker. It's clear what you mean, so no need to change the code.